hermes-api/src/main/java/pl/allegro/tech/hermes/api/MaintainerDescriptor.java

```diff
@@ -0,0 +1,53 @@
+package pl.allegro.tech.hermes.api;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.validation.constraints.NotNull;
+import java.util.Objects;
+
+public final class MaintainerDescriptor {
```

Comment:
So if it is connected with persistence/storage why its in  `pl.allegro.tech.hermes.api`?